network-uri-json.cabal

```diff
@@ -1,5 +1,5 @@
 name:                network-uri-json
-version:             0.3.1.1
+version:             0.3.1.2
```

Comment:
I use git flow to manage these versions so I'll bump this version when I cut the release based on the included changes.  You can go ahead and remove this from this diff.  Once this passes tests and lands I'll create the release branch.

Reply:
Done!

Reply:
Thanks!